src/hearings/containers/request-hearing/hearing-requirements/hearing-requirements.component.ts

```diff
@@ -143,13 +147,14 @@ export class HearingRequirementsComponent extends RequestHearingPageFlow impleme
 
   public combinePartiesWithIndOrOrg(partyDetails: PartyDetailsModel[]): PartyDetailsModel[] {
     const combinedPartyDetails: PartyDetailsModel[] = [];
+    const flagsForParty: string[] = []; 
     partyDetails.forEach((partyDetail) => {
       const organisationDetails = partyDetail.organisationDetails;
       const party: PartyDetailsModel = {
         ...partyDetail,
         individualDetails: {
           ...partyDetail.individualDetails,
-          reasonableAdjustments: this.getAllPartyFlagsByPartyId(partyDetail.partyID)
```

Comment:
We don't need any business logic, should be fine to just write
reasonableAdjustments: partyDetail.?individualdetails.?reasonableAdjustments